fdb-relational-core/src/main/java/com/apple/foundationdb/relational/util/NullableArrayUtils.java

```diff
@@ -20,9 +20,9 @@
 
 package com.apple.foundationdb.relational.util;
 
-import com.apple.foundationdb.annotation.API;
+import static com.apple.foundationdb.record.expressions.RecordKeyExpressionProto.*;
```

Comment:
use static import to make file more readable.

Reply:
checkstyle doesn't allow "*" in imports, reverted the change.

Reply:
You can static import them separately. Sometimes, IntelliJ is configured to group them into a `*` import.

Reply:
right, but unfortunately build still fails because checkStyle doesn't allow * in import lines.